src/components/victory-line.jsx

```diff
@@ -89,6 +90,11 @@ export default class VictoryLine extends React.Component {
      */
     label: React.PropTypes.string,
     /**
+     * The labelComponent prop takes in an entire, HTML-complete label component
+     * which will be used to create labels for line to use
+     */
+    labelComponent: React.PropTypes.element,
+    /**
```

Comment:
Ah I only meant that "div" should be possible if we kept `labelComponent` as a prop, in which case there's no ambiguity as to whether it's the label itself or what element type to use (`labelComponent` would always be a type, never an instance).

I'll check out VictoryBar and see if there's anything we could improve there as well.